LibVLCSharp/Shared/MediaPlayer.cs

```diff
@@ -631,6 +631,11 @@ struct Native
 #endif
         }
 
+        public void SetVideoFormatCallbacks()
+        {
+            throw new NotImplementedException();
```

Comment:
What's up with this? :D https://github.com/Ch4rg3r/libvlcsharp/blob/7ddc025392caf39ec1277c4f59585b672d88cf8b/LibVLCSharp/Shared/MediaPlayer.cs#L1272